tests/zfs-tests/tests/functional/migration/migration_004_pos.ksh

```diff
@@ -61,7 +61,7 @@ cd $DNAME
 (( $? != 0 )) && log_untested "Could not change directory to $DNAME"
 
 $LS $BNAME | $CPIO -oc > $TESTDIR/cpio$$.cpio
-(( $? != 0 )) && log_failED "Unable to create cpio archive"
+(( $? != 0 )) && log_fail "Unable to create cpio archive"
```

Comment:
@behlendorf Thanks for your suggestion. You are a careful man, the same typo has been solved.  And I'm reviewing the migration tests.